app/models/public_body.rb

```diff
@@ -872,10 +872,14 @@ def update_counter_cache
       ['info_requests_visible_count', {}]
     ]
 
-    mappings.each do |column, extra_params|
-      params = basic_params.clone.update extra_params
-      update_column(column, InfoRequest.where(params).is_searchable.count)
+    updated_counts = mappings.reduce({}) do |memo, mapping|
+      column = mapping.first
+      params = basic_params.clone.update(mapping.last)
+      memo[column] = InfoRequest.where(params).is_searchable.count
+      memo
```

Comment:
I've always just used `memo` because they use it in [Ruby Doc](http://ruby-doc.org/core-2.5.0/Enumerable.html#method-i-inject).

IDK if there's a particular cop for this.

-1 to `h` or `a` as they don't [communicate well](http://rubocop.readthedocs.io/en/latest/cops_naming/#naminguncommunicativeblockparamname).

`each_with_object` seems like a better fit anyway

Reply:
Moved to `each_with_object` in d3bd013ee. Not addressing `memo` as we don't have a specific cop for it.